ros2_batch_job/__init__.py

```diff
@@ -1 +0,0 @@
-DEFAULT_REPOS_URL = 'https://raw.githubusercontent.com/ros2/ros2/master/ros2.repos'
```

Comment:
See second paragraph after the green blob in the docs: https://docs.python.org/3/tutorial/modules.html#packages

Therefore keeping the empty file.